src/txn/txn_timestamp.c

```diff
@@ -705,6 +738,28 @@ __wt_txn_set_timestamp(WT_SESSION_IMPL *session, const char *cfg[])
 	return (0);
 }
 
+/*
+ * __wt_txn_query_timestamp --
+ *	Query a timestamp.
```

Comment:
Add same addition as internal function: `Query a timestamp within this session's txn.`